sematic/ee/LICENSE

```diff
@@ -0,0 +1,36 @@
+The Sematic Enterprise license (the "Enterprise License")
+Copyright (c) 2023 - present Sematic Inc.
+
+With regard to the Sematic Software:
+
+This software and associated documentation files (the "Software") may only be
+used in production, if you (and any entity that you represent) have agreed to,
+and are in compliance with, the Sematic Enterprise Terms of Service, available
+via email (emmanuel@sematic.dev) (the "Enterprise Terms"), or other
+agreement governing the use of the Software, as agreed by you and Sematic,
+and otherwise have a valid Sematic Enterprise license for the
+correct usage volume. Subject to the foregoing sentence, you are free to
+modify this Software and publish patches to the Software. You agree that Sematic
+and/or its licensors (as applicable) retain all right, title and interest in and
```

Comment:
Is this from the original formulation? Seems wrong. Is it legalese? (There's another instance below in the text as well)
```suggestion
and/or its licensors (as applicable) retain all rights, titles and interests in and
```